### ISSUE TITLE
Missing WCAG link

### ISSUE THREAD

**Issue by ferBonnin** [Microsoft/accessibility-insights-web#507]
## Describe the bug

**To Reproduce**

1. Go to any page
2. Run Accessibility Insights for Web > Assessment
3. Go to Multimedia > Audio Description > more info and examples
4. missing link for WCAG 1.2.3

## Expected behavior

Add a link in "WCAG Success criteria" for this requirement.
Name:  Understanding Success Criterion 1.2.3: Audio Description or Media Alternative (Prerecorded)
link: https://www.w3.org/WAI/WCAG21/Understanding/audio-description-or-media-alternative-prerecorded

## Screenshots
.
![image](https://user-images.githubusercontent.com/9062104/55745266-af9d0280-59eb-11e9-8b8b-b36f6722f691.png)

## Extension (please complete the following information)

NA

## Are you willing to submit a PR?

If asked, will you be willing to submit a PR to fix this bug? No

## Did you search for similar existing issues?
Yes

**Comment by ferBonnin** [comment]
@LiLoDavis I do not think there needs to be more links for sufficient techniques or additional guidance?

**Comment by LiLoDavis** [comment]
@ferBonnin Agreed.
